service/prometheus/scrapeconfig.go

```diff
@@ -46,9 +50,10 @@ func GetScrapeConfigs(services []v1.Service, certificateDirectory string) ([]con
 			Scheme:  httpsScheme,
 			HTTPClientConfig: config.HTTPClientConfig{
 				TLSConfig: config.TLSConfig{
-					CAFile:   key.CAPath(certificateDirectory, clusterID),
-					CertFile: key.CrtPath(certificateDirectory, clusterID),
-					KeyFile:  key.KeyPath(certificateDirectory, clusterID),
+					CAFile:             key.CAPath(certificateDirectory, clusterID),
+					CertFile:           key.CrtPath(certificateDirectory, clusterID),
+					KeyFile:            key.KeyPath(certificateDirectory, clusterID),
+					InsecureSkipVerify: true,
```

Comment:
We need insecure skip verify here to get over the fact workers don't have correct IP sans. We can remove this in the future.

Reply:
How are we going to fix this?

Reply:
See https://github.com/giantswarm/giantswarm/issues/2015 for tracking separate certificates for each worker.